### ISSUE TITLE
Consider cropping samples to a fixed maximum length

### ISSUE THREAD

**Issue by ibab** [ibab/tensorflow-wavenet#19]
It's not ideal that the length of the samples varies wildly.
This could be fixed by cropping (or even padding) them to a fixed size.

**Comment by genekogan** [comment]
another option would be to split files into fixed-size samples of `n` seconds. then you don't lose training data from cropping.

**Comment by ibab** [comment]
This is fixed by #54.
